zone_481.geojson

```diff
@@ -0,0 +1 @@
+{"type": "MultiPolygon", "coordinates": [[[[1546511.2076999992, 573564.3313999996], [1548398.1775000002, 571624.9122000001], [1546443.8325999994, 568942.5441999994], [1547537.3745000008, 563893.9375], [1552857.1240999997, 563469.3748000003], [1556480.1875999998, 551723.2838000003], [1556179.5780999996, 548095.2435999997], [1559016.2506000008, 546669.1884000003], [1560228.2488000002, 535774.0618999992], [1557941.2512999997, 527584.7489], [1560952.0489000008, 522752.01359999925], [1559850.0938000008, 519946.5362], [1562023.4190999996, 513684.36659999937], [1569337.6259000003, 501371.4056000002], [1571276.3732999992, 505303.7335000001], [1576472.2189000007, 500275.07840000093], [1578417.8761999998, 500493.37480000034], [1580256.2511999998, 492387.06310000084], [1578936.9049999993, 491161.4743000008], [1580291.1368000004, 475220.68200000003], [1493724.138699999, 466194.81010000035], [1492315.0778, 472520.7011999991], [1489446.8088000007, 472247.59219999984], [1490412.5431999993, 474298.2587000001], [1489229.6675000004, 474922.64880000055], [1491038.0473999996, 477148.8370999992], [1489873.5463999994, 481174.50879999995], [1493770.6037000008, 487918.35569999926], [1496565.0219, 489198.6956999991], [1496716.0659999996, 490694.58359999955], [1492058.5166999996, 492762.3917999994], [1492866.2109999992, 497281.2301000003], [1495946.0834999997, 501021.87340000086], [1493328.0923999995, 502254.1754999999], [1493112.2577, 504493.19659999944], [1489377.6642000005, 504435.3856000006], [1490427.8293999992, 507776.38230000064], [1488695.8932000007, 508968.95720000006], [1490583.8584000003, 510470.6532000005], [1492355.0956999995, 518334.68239999935], [1491308.2809999995, 521066.0795000009], [1495446.6787, 530848.7774999999], [1496152.5711000003, 537408.5092999991], [1500272.3279, 539708.9989999998], [1501591.5471, 549895.0096000005], [1509333.200099999, 547306.9069999997], [1510532.9472000003, 548308.9236999992], [1509936.0902999993, 553217.8772999998], [1512997.4448000006, 556661.7409000006], [1519426.6698000003, 556319.3234999999], [1519304.9940000009, 565655.2257000003], [1522646.9327000007, 565167.1259000003], [1524305.0427, 561652.6063000001], [1527038.6434000004, 561222.5099], [1531073.3099000007, 569889.8015999999], [1535986.6667, 568966.9482000005], [1536642.7149, 570605.8478999995], [1538782.1048000008, 570153.4283000007], [1539002.4859999996, 572229.4941000007], [1546511.2076999992, 573564.3313999996]]]], "attributes": {"MOF_FIRE_ZONE_ID": 481.0, "MOF_FIRE_CENTRE_NAME": "Southeast Fire Centre", "MOF_FIRE_ZONE_NAME": "Boundary Fire Zone", "HEADQUARTERS_CITY_NAME": "Grand Forks", "OBJECTID": 1092, "FEATURE_AREA_SQM": 6590044104.883, "FEATURE_LENGTH_M": 457055.5345, "GEOMETRY.AREA": 0, "GEOMETRY.LEN": 0}}
```

Comment:
I think these are all currently in the root dir of our repo. 
Also, might not be worth the effort, but all these geojsons could be a single file & iterate through features

Reply:
Oops thanks I'll move those. I'll defer merging them into a single file for now unless it becomes a problem.

Reply:
I'll also delete the deployment and crunchydb cluster so it reruns migrations